android/build.gradle

```
@@ -27,10 +27,10 @@ apply plugin: 'kotlinx-serialization'
 
 android {
     namespace "com.aheaditec.freerasp"
-    compileSdkVersion project.hasProperty('compileSdkVersion') ? rootProject.ext.compileSdkVersion : 33
+    compileSdkVersion project.hasProperty('compileSdkVersion') ? rootProject.ext.compileSdkVersion : 35
```

```suggestion
    compileSdk 35
```

compileSdkVersion is deprecated, use compileSdk instead. Also the version must now be hardcoded (here it was taken from the project settings first, 35 is only fallback)